sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_generated/operations/_schema_operations.py

```diff
@@ -63,6 +63,7 @@ def get_by_id(
         cls = kwargs.pop('cls', None)  # type: ClsType[str]
         error_map = {404: ResourceNotFoundError, 409: ResourceExistsError}
         error_map.update(kwargs.pop('error_map', {}))
+        api_version = "2017-04"  # TODO: manually patch, the default value generated is "2018-01-01-preview"
```

Comment:
We shall discuss that, I'm against any kind of manual patching of generated code, doesn't scale

Reply:
according to @arerlend , new api version is still being worked on.